ww3tools/modelBuoy_collocation.py

```diff
@@ -55,6 +56,26 @@
   multiple forecast data files:
   nohup python3 modelBuoy_collocation.py ww3list.gfs-d36.GSE1.5.txt 2 gridInfo.nc CycloneMap_2021.nc >> nohup_modelBuoy_collocation.out 2>&1 &
```

Comment:
@Ghazal-Mohammadpour Seems like with you new code, this line will not run without error.